test/integration/features/index.js

```diff
@@ -1082,13 +1082,23 @@ describe('features', () => {
         bobRegTxId = result;
 
         bobPreviousST = result;
+
+        //now we verify that getUTXO.items not empty when no new block generated
+        const utxo = await dapiClient.getUTXO(faucetAddress);
+        console.log(utxo);
```

Comment:
Please remove console logs

Reply:
I log only this case to show value of getUtxo for existing bug to track this value

Reply:
But if it's only for debug purposes it should not go to the master then.